### DISCUSSION TITLE
Rumble not working with genuine DS3 on XInput mode

### DISCUSSION THREAD

**Discussion by TheFlatsman** [nefarius/DsHidMini#123] [Support]
Im running version 2.0.248.0 on Win10 and as mentioned on the title Rumble doesn't work on XInput mode via USB neither Bluetooth, but when testing with MightyHIDTest on SXS mode it works fine on both connection types.

PS: Worked fine with previous versions of Shibari, the latest one's had a delay so rumble would kick in 2 or 3 seconds after the fight and keep active for some seconds so i had to disable vibration on the config.

**Comment by inibintang** [comment]
have you solve it? i have the same problem, but mine work with usb
